src/lib/elementary/Elementary.h

```diff
@@ -89,18 +91,18 @@
 #ifdef _WIN32
 # ifdef EFL_BUILD
```

Comment:
Should it use `EFL_BUILD`? I think this whole EAPI thing here is already defined at the above included `elementary_api.h`.

Reply:
This probably should include `elementary_api.h` instead of redefining it.